crates/trycmd/Cargo.toml

```diff
@@ -55,8 +55,7 @@ rayon = "1.5.1"
 
 serde = { version = "1.0", features = ["derive"] }
 shlex = "1.1.0"
-humantime = "2"
-humantime-serde = "1"
+jiff = { version = "0.2.4", features = ["serde"] }
```

Comment:
Should we disable default features?  Doesn't seem like we're using time zone functionality

Reply:
yep doing it

Reply:
I removed the `serde` feature altogether as json schema is not implemented, and it's easier to add deserialize than implementing the json schema interface for `jiff`

Reply:
You can do something like
```rust
    #[cfg_attr(feature = "unstable-schema", schemars(with = "Option<std::time::Duration>"))]
```
(we do this a lot in `cargo-util-schemas`)

Reply:
i guess this means adding the implementation in `jiff`, not sure this is first goal as it's not 1.0.0 yet

Reply:
`jiff` does not need to have native support for `shemars` (and it would be a bad idea to do so).

THe attribute I mentioned above is a way to say "use the schema of this other type".

Reply:
Looking into it